content/applications/marketing/marketing_automation/advanced/understanding_metrics.rst

```diff
@@ -3,78 +3,83 @@ Understanding metrics
-previously-specified criteria filter. In this example, there are 18 records in the database that
-match the criteria.
+Beneath the filter nodes, Odoo displays how many records in the database fall into the
+previously-specified criteria filter.
+
+In this example, there are 18 records in the database that match the criteria.
 
 .. image:: understanding_metrics/marketing-automation-lead-filters.png
    :align: center
    :alt: Lead generation filters used to refine records on marketing automation campaigns in Odoo.
 
-At the top of the marketing automation form, there are a series of smart buttons. The
+At the top of the marketing automation campaign form, there are a series of smart buttons. The
 :guilabel:`Participations` smart button in this example indicates that, out of those 18 records, 25
 have become participants. In other words, they have matched the criteria, and have already
 interacted with the campaign in one way or another.
 
 .. note::
    :guilabel:`Records` represent a real-time number. Therefore, while the workflow is running,
-   changes in opportunity records (deletions, additions, adjustments) may occur. This means there
-   will be real-time updates to the number of :guilabel:`Records`, but such updates do *not* change
-   the number of :guilabel:`Participants`, as the metric *will not* exclude opportunities that have
-   already been set as :guilabel:`Participants` before - Odoo will just add new ones. That's why
-   the number of :guilabel:`Records` can be different from the number of :guilabel:`Participants`
-   from time to time.
+   changes in opportunity records (deletions, additions, or adjustments) may occur.
+
+   This means there will be real-time updates to the number of :guilabel:`Records`, but such updates
+   do *not* change the number of :guilabel:`Participants`, as the metric *will not* exclude
+   opportunities that have already been set as :guilabel:`Participants` before - Odoo just adds new
+   ones.
+
+   That's why the number of :guilabel:`Records` can be different from the number of
+   :guilabel:`Participants` from time to time.
 
 .. image:: understanding_metrics/campaign-records-vs-participants.png
    :align: center
    :alt: Difference between real-time record count and total participants on a markeing campaign.
 
 Whenever the system runs, updating numbers and triggering actions, it will look at the
-:guilabel:`Target` model, and check if new :guilabel:`Records` have been added (or modified),
-which always keeps the flow up-to-date.
+:guilabel:`Target` model, and check if new :guilabel:`Records` have been added (or modified), which
+always keeps the flow up-to-date.
 
 .. note::
-   The filter here is applied to *all* activities. Leads that lose that specific :guilabel:`Tag`
-   in the meantime will be excluded from later activities.
+   The filter here is applied to *all* activities. Leads that lose that specific :guilabel:`Tag` in
```

Comment:
"The filter here" relies on the image for context, can you please specify exactly which filter is being referred to?